contracts/Ownable.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity ^0.5.0;
+
```

Comment:
It would have been good that you experiment DIY before picking it from OpenZepellin.